src/main/java/com/example/demo/controller/PdfandwordtoexcelController.java

```diff
@@ -0,0 +1,61 @@
+package com.example.demo.controller;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import com.example.demo.Service.PdfandwordtoexcelService;
+
+@RestController
+public class PdfandwordtoexcelController {
+
+	@Autowired
+	private PdfandwordtoexcelService pdfandwordtoexcelService; 
+	
+	@PostMapping("/getExcelByPdf")
+	public ResponseEntity<byte[]> getExcelByPdf(
+			@RequestParam("pdfWordFile") List<MultipartFile> pdfFiles) {
+
+		HttpHeaders headers = new HttpHeaders();
```

Comment:
Why don't you @Autowried for HttpHeader class and use for entire class. So that the length of code will reduce and will follow the coding standards.

Reply:
Yes Vinay-1998 I changed to @Autowired